### ISSUE TITLE
Disable Flushing between prints when not changing filament

### ISSUE THREAD

**Issue by jdpg2** [SoftFever/OrcaSlicer#7319]
### Is there an existing issue for this feature request?

- [X] I have searched the existing issues

### Is your feature request related to a problem?

It would be good to have the option to NOT flush between prints when no filament change is needed or when the AMS is not in use.  Right now i have several printer profiles with G Code modifications and its inconvenient to proceed that way.  Additionally some of my filament is highly costly engineering filament and every little bit counts.

### Which printers will be beneficial to this feature?

Others

### Describe the solution you'd like

Would be nice if we were presented with an option when we send the slice to the printer much like use AMS or Bed leveling.  A check box that says "Disable Flushing" would be ideal obvious with the stated caveats that the current color will be used etc....  This could be only enabled with AMS disabled as a start?

### Describe alternatives you've considered

Multiple printer profiles which is tedious when you have different profiles to change z off sets based on materials as well.

**Comment by hearthburst** [comment]
yeah, it would also reduce the amount of poop filament everywhere too
